auth0/src/test/java/com/auth0/android/authentication/storage/CredentialsManagerTest.java

```diff
@@ -319,4 +321,68 @@ public void shouldGetAndFailToRenewExpiredCredentials() throws Exception {
         assertThat(exception.getCause(), Is.<Throwable>is(authenticationException));
         assertThat(exception.getMessage(), is("An error occurred while trying to use the Refresh Token to renew the Credentials."));
     }
+
+    @Test
+    public void shouldClearCredentials() throws Exception {
+        manager.clearCredentials();
+
+        verify(storage).remove("com.auth0.id_token");
+        verify(storage).remove("com.auth0.access_token");
+        verify(storage).remove("com.auth0.refresh_token");
+        verify(storage).remove("com.auth0.token_type");
+        verify(storage).remove("com.auth0.expires_at");
+        verify(storage).remove("com.auth0.scope");
```

Comment:
just to avoid forgetting checks in the future (supposed you add a new key) you can also verify for "no more interactions" in storage